### DISCUSSION TITLE
Problem when using the plugin with Azure DevOps Pipeline

### DISCUSSION THREAD

**Discussion by bohrz** [scolladon/sfdx-git-delta#613] [Q&A]
Hi!

I'm trying to use the plugin in an Azure DevOps pipeline (similar to GitHub actions), which I have little experience. What I'm doing:

1. Get the target branch with $(System.PullRequest.TargetBranch), which returns 'refs/heads/target-branch'
2. Replace refs/heads/ with empty string, which returns 'target-branch'
3. Repeat steps 1 and 2 for the source branch
4. Run the command `sfdx sgd:source:delta --to $(target-branch) --from $(source-branch) --output ./packages`
5. The command erros out:
```
{
  "error": "fatal: Not a valid object name test/devops\n",
  "output": "./packages",
  "success": false,
  "warnings": []
}
```
I don't know why the variable has the new-line character at the end and it is messing the command execution. Anyone experienced similar problem or have any suggestions?

I'm searching Azure DevOps forums and documentation as well, but not making any progress.

**Comment by scolladon** [comment]
Hi @bohrz,

I don't know why there is a `new-line` char in this case.
I don't have much experience with azure devops.

We have a section dedicated to [CI/CD](https://github.com/scolladon/sfdx-git-delta#cicd-specificity) and [Windows](https://github.com/scolladon/sfdx-git-delta#windows-users) in the doc.

Have you tried stripping out those `new-line`?
